app/views/browse/build.html

```diff
@@ -10,10 +10,11 @@ <h1 class="contains-actions">
             <!-- Primary Actions -->
             <button class="btn btn-default hidden-xs"
                     ng-click="cancelBuild()"
-                    ng-if="!build.metadata.deletionTimestamp && (build | isIncompleteBuild) && ('builds' | canI : 'update')">Cancel Build</button>
+                    ng-if="!build.metadata.deletionTimestamp && (build | isIncompleteBuild) && ('builds' | preferredVersion | canI : 'update')">Cancel Build</button>
```

Comment:
I think I'm not a fan of using the filter unless necessary.  This creates a lot of template noise.  Going to update to use `$scope.buildVersion` instead.